### PR TITLE
Update console example output.

### PR DESCRIPTION
**Pull request by bcrowe**

### CODE REVIEW

File: en/console-and-shells.rst [hunk 1]
```diff
@@ -67,7 +67,11 @@ Running the Console with no arguments produces this help message::
 
     Available Shells:
 
-    [CORE] bake, i18n, server, test
+    [Bake] bake
+
+    [Migrations] migrations
+
+    [CORE] i18n, server, plugin, server
```
Comment (antograssiot) [inline en/console-and-shells.rst]: @bcrowe you mention ``server`` twice and missed ``orm_cache``

Comment (bcrowe) [inline en/console-and-shells.rst]: Doh!

**Comment by antograssiot** [comment]
we probably could already update 3.1 and add ``routes``. @bcrowe are you up to that ?

**Comment by bcrowe** [comment]
@antograssiot You know it!

**Pull request merged**